octobot_trading/exchanges/abstract_websocket_exchange.py

```diff
@@ -32,6 +32,9 @@ class AbstractWebsocketExchange:
 
     INIT_REQUIRING_EXCHANGE_FEEDS = set()
 
+    # Used to ignore a feed when almost one of the corresponding feed is supported
```

Comment:
# Used to ignore a feed when **at least** one of the corresponding feed is supported

?

Reply:
Right :smile: